code/solr/SolrIndex.php

```diff
@@ -472,14 +492,18 @@ function commit() {
 	 * @param SearchQuery $query
 	 * @param integer $offset
 	 * @param integer $limit
-	 * @param  Array $params Extra request parameters passed through to Solr
+	 * @param array $params Extra request parameters passed through to Solr
 	 * @return ArrayData Map with the following keys: 
 	 *  - 'Matches': ArrayList of the matched object instances
 	 */
 	public function search(SearchQuery $query, $offset = -1, $limit = -1, $params = array()) {
 		$service = $this->getService();
 		
-		SearchVariant::with(count($query->classes) == 1 ? $query->classes[0]['class'] : null)->call('alterQuery', $query, $this);
+		SearchVariant::with(
+			count($query->classes) == 1
```

Comment:
Can we break this out to a separate statement? I don't like trinaries if we have to break them over so many lines like this.